app/views/homes/show.html.erb

```diff
@@ -45,7 +45,7 @@
         in our industry.
       </p>
 
-      <%= link_to "Learn more about us", "/about.html", :class => "to-about" %>
+      <%= link_to "Learn more about us", page_path("about"), :class => "to-about" %>
```

Comment:
While you're editing this line anyway, could you change the hash syntax to the newer version? i.e. `class: "to-about"`.